R/expect-chk-error.R

```diff
@@ -0,0 +1,19 @@
+#' Expect Chk Error
+#'
+#' [expect_chk_error()] checks that code throws an error
+#' of class `"chk_error"` with a message that matches regexp.
+#' See below for more details.
+#'
+#' @inherit testthat::expect_error
+#' @param class Must be `"chk_error"`.
+#' @export
+#'
+#' @examples
+#' expect_chk_error(chk_true(FALSE))
+#' try(expect_chk_error(chk_false(FALSE)))
+expect_chk_error <- function(object, regexp = NULL, class = "chk_error", ..., info = NULL, label = NULL) {
```

Comment:
```suggestion
expect_chk_error <- function(object, regexp = NULL, ..., info = NULL, label = NULL, class = NULL) {
```

and check if it's `NULL`.